docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,331 @@
+# Autofill Service
+
+Autofill Service application is a service application to store the user input and provides the data previously entered by the user. Developers can implement the Autofill service in their own application to serve the Autofill information.
+
+The main features of the Autofill service API include:
+
+- Receive Authentication Information
+
+  You can [receive the request of authentication, and send the authentication information](#receive-authentication-information).
+
+- Receive Fill Request and Send Fill Response
+
+  You can [receive the fill request, and send the fill response](#receive-fill-request-and-send-fill-response).
+
+- Receive Request of Saving Autofill Data
+
+  You can [receive the request of saving the autofill data](#receive-request-of-saving-autofill-data).
+
+## Prerequisites
+
+1. To use the Autofill Service API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__AUTOFILL__SERVICE_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__AUTOFILL__SERVICE_MODULE.html) applications), the application must have metadata key in the `tizen-manifest.xml` file to be recognized as an Autofill Service:
```

Comment:
Please check the link.
It looks like the actual link is "group__CAPI__UIX__AUTOFILL__SERVICE__MODULE.html"
Not a link with one _ missed "group__CAPI__UIX__AUTOFILL__SERVICE_MODULE.html"

Reply:
Fixed.